src/staking/validation_error.cpp

```diff
@@ -91,6 +99,58 @@ const ValidationError &GetValidationErrorFor(const staking::BlockValidationError
       static ValidationError err("bad-blk-public-key");
       return err;
     }
+    case staking::BlockValidationError::INVALID_BLOCK_WEIGHT: {
+      static ValidationError err("bad-blk-weight");
+      return err;
+    }
+    case staking::BlockValidationError::INVALID_BLOCK_SIGOPS_COUNT: {
+      static ValidationError err("bad-blk-sigops");
+      return err;
+    }
+    case staking::BlockValidationError::INVALID_COINBASE_LENGTH: {
```

Comment:
I could not find any usages of `INVALID_COINBASE_LENGTH` except this one

Reply:
Removed in 2681ce3397ef2754e00e0598e194f639d95219da